portfolio.html

```diff
@@ -54,5 +54,113 @@ <h2>I am a Software Developer</h2>
+      <div id="set2">
+        <h2>Multi-Post Stories</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+      </div>
+      <div id="lang">
+        <img src="/images/Icons/languages/css.png" alt="csslang" />
+        <img src="/images/Icons/languages/html.png" alt="htmllang" />
+        <img src="/images/Icons/languages/bootstrap.png" alt="bootlang" />
+        <img src="/images/Icons/languages/ruby.png" alt="rubylang" />
+      </div>
+      <div class="project-button">
+        <a href="projectlink"
+          ><img
+            src="/images/Icons/Button/Enable.png"
+            alt="button
+        icon"
+        /></a>
+      </div>
+    </section>
+    <section class="project-works">
+      <div class="smallworks">
+        <h1>Professional Art Printing Data</h1>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <div>
+          <img
+            src="/images/Icons/languages/darkicons/html.png"
+            alt="htmllang"
+          />
+          <img
+            src="/images/Icons/languages/darkicons/bootstrap.png"
+            alt="bootlang"
+          />
+          <img
+            src="/images/Icons/languages/darkicons/ruby.png"
+            alt="rubylang"
+          />
+        </div>
+      </div>
+      <div class="longbutton">
+        <a href="#longbotton"><img src="/images/Action.png" alt="button" /> </a>
+      </div>
+    </section>
+    <section class="about-section">
+      <div class="about">
+        <h1>About me</h1>
+
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <div id="resume">
+          <a href="#getmyres"
+            ><img
+              class="resbutton"
+              src="/images/Icons/Button/resume/Enable.png"
+              alt="resumeicon"
+          /></a>
+        </div>
+      </div>
+    </section>
+    <section class="about-lang">
+      <div>
+        <hr class="solid" />
+      </div>
+      <div class="learn">
+        <h1><span>&#9830;</span> Languages</h1>
+        <h2>Javascript</h2>
+        <h2>Ruby</h2>
+        <h2>Html</h2>
+        <h2>Css</h2>
+        <h1><span>&#9632;</span> Frameworks</h1>
+        <h2>Bootstrap</h2>
+        <h2>Ruby on Rails</h2>
+        <h2>RSPec</h2>
+        <h2>CapyBara</h2>
+        <h2>Selenium</h2>
+        <h1><span>&#9679;</span> Skills</h1>
+        <h2>Codekit</h2>
+        <h2>Github</h2>
+        <h2>Codepen</h2>
+        <h2>Gitlab</h2>
+        <h2>Terminal</h2>
```

Comment:
The categories inside the about myself section should be inside a `ul` tag, and each one of the elements ( Languages, Frameworks, and Skills ) should be in `li` tag as stated in the requirements, Kindly fix this 👍😃